src/odemis/gui/win/acquisition.py

```diff
@@ -1067,6 +1071,7 @@ def update_acquisition_time(self):
             self.lbl_acqestimate.SetLabel("Add a stream area to acquire.")
             return
 
+        focus_points_dist = self.focus_points_dist_ctrl.GetValue()
```

Comment:
```suggestion
        focus_points_dist = self.focus_points_dist.value
```

Reply:
You have to update the object! `focus_points_dist_ctrl` -> `focus_points_dist`

Reply:
My bad, the VA works. I have updated the PR